### PR TITLE
parametrisation completed and HPC option working on EDDIE

### PR DESCRIPTION
**Pull request by MazdaX**
This is a major upgrade of the pipeline from the last fork to BovReg/nf-cage.

### CODE REVIEW

File: [:]/trace.txt [hunk 1]
```diff
@@ -0,0 +1,70 @@
+task_id	hash	native_id	name	status	exit	submit	duration	realtime	%cpu	peak_rss	peak_vmem	rchar	wchar
```
Comment (JoseEspinosa) [inline [:]/trace.txt]: Did you want to include this file or is it a leak?

**Review by JoseEspinosa — APPROVED** [review]
Very nice @MazdaX 😄 
Just was not sure if there is a file that was commit by mistake but otherwise looks awesome 🚀

File: [:]/trace.txt (hunk 1 above)
Comment (MazdaX) [reply [:]/trace.txt]: I don't know why it keeps being pushed its in the .gitignore:
https://github.com/MazdaX/nf-cage/blob/4ebad885f695396b4c0e81cf836a90f9645bf3a7/.gitignore#L31
I must have missed it during the conflict resolution step. It happens I guess ;-)

**Review by JoseEspinosa — APPROVED** [review]
I'll then merge the PR, thanks @MazdaX 🚀

**Pull request merged**